examples/generic_connector_configs/README.md

```diff
@@ -219,147 +240,140 @@ The configuration file for the Instagram API
 
 <details>
   <summary><b>Example setup to try this out locally</b></summary>
-  This example focuses on Instagram, but using Secretless with the
-   rest of the Facebook API's is the same from Step 2 on.
+  This example focuses on Instagram, but using Secretless with the rest of the
+  Facebook API's is the same from Step 2 on.
 
   1. Follow this guide to set up an
    [example Instagram app](https://developers.facebook.com/docs/instagram-basic-display-api/getting-started)
    and acquire an access token
-
-  2. Store the token in your local credential manager so
+  1. Store the token in your local credential manager so
    that it may be retrieved in your `secretless.yml`
-
-  3. Build and run Secretless locally
-
-  ```
-  ./bin/build_darwin
-  ./dist/darwin/amd64/secretless-broker \
-  -f examples/generic_connector_configs/instagram_secretless.yml
-  ```
-
-  4. In another terminal window, make a request to Instagram using Secretless
-
-  ```
-      http_proxy=localhost:8023 curl -X GET --url 'graph.instagram.com/me?fields=id,username'
-  ```
+  1. Build and run Secretless locally
+     ```
+       ./bin/build_darwin
+       ./dist/darwin/amd64/secretless-broker \
+       -f examples/generic_connector_configs/instagram_secretless.yml
+     ```
+  1. In another terminal window, make a request to Instagram using Secretless
+     ```
+        http_proxy=localhost:8023 curl -X GET --url 'graph.instagram.com/me?fields=id,username'
+     ```
 </details>
 
 ___
 
 ### GitHub API
-This example can be used to interact with [GitHub's API](https://developer.github.com/v3/).
 
-The configuration file for the GitHub API can be found at [github_secretless.yml](./github_secretless.yml).
+This example can be used to interact with
+[GitHub's API](https://developer.github.com/v3/).
```

Comment:
Would be clearer to include the version in the hyperlink, e.g. show `GitHub's API v3` rather than `GitHub's API`? Looks like you've excluded the version from similar links throughout this README.md, so you may have a reason or preference for leaving this out... and I certainly can go either way on this.

Reply:
I'm going to just use [developer.github.com](https://developer.github.com/)!